### PR TITLE
Update UG, DG and PPP

### PR DESCRIPTION
**Pull request by seanpzk**
UG:
- Added `unmarkall /c` command
- Fix grammar

DG:
- Added use case for Start a Lesson
- Add unmarkall feature under manual testing section
- Fix grammar

PPP:
- Tweaked phrasing

### CODE REVIEW

**Review by DonovanJJ — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -442,7 +443,7 @@ startlesson /c CS2103T
 
 <br>
 
-### Setting number of lessons of a class : `setlesson /c`
+### Setting the number of lessons of a class : `setlesson /c`
```
Comment (songgthu) [inline docs/UserGuide.md:459]: Can you update this in the table of content and its hyperlink too thanks!

**Review by songgthu — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (seanpzk) [reply docs/UserGuide.md:459]: Okay!

**Pull request merged**